include/guiheaderfiles/customerdialog.h

```diff
@@ -0,0 +1,44 @@
+#ifndef CUSTOMERDIALOG_H
+#define CUSTOMERDIALOG_H
+
+#include <QDialog>
+#include <QTimer>
+#include <string>
+#include <vector>
+using namespace std;
```

Comment:
adding "using namespace std" isn't recommended. Remove this and add std::string, etc.